core/src/main/java/org/jdbi/v3/core/argument/Arguments.java

```diff
@@ -66,6 +66,7 @@ public void setRegistry(ConfigRegistry registry) {
     private Arguments(Arguments that) {
         factories.addAll(that.factories);
         untypedNullArgument = that.untypedNullArgument;
+        registry = null;
```

Comment:
What you do manually/on sight, I let my IDE do for me:
![image](https://user-images.githubusercontent.com/3950300/60628710-fa63c600-9df2-11e9-8d23-88c01142dcd6.png)

Don't set `registry` to null -> you learn to ignore the "missing a field" warning, so when you add a fiield and really do forget to copy it, you won't notice. Shooting yourself in the foot twice.

Reply:
Functionally pointless as it may be, forcing a copy constructor to deal with all fields to live up to its design concept adds clarity to code, and allows IDEs to be more helpful.

Reply:
Since registry should only be set in the `setRegistry()` method, I think it's ok to explicitly set it to null on the field initializer.

Reply:
I can leave it as defult (null), but the copy constructor should assign null to avoid a false-positive "copy constructor misses a field", which is a helpful warning I'd prefer to enable.

Reply:
I'm rather -1 to pointless null initializations.  It's hard to see it as anything but noise in the code.

Reply:
So I'm getting a vibe from both of you that you don't really care about copy constructors violating the contract of a copy constructor due to implementation details then